### PR TITLE
Arcana login updated sdk integration

### PR DESCRIPTION
**Pull request by shrinathprabhu**
1. Removed unwanted dependencies
2. Renamed signer-make-tx to tx-sign. This file is needed for making smart contract calls for updating config in dashboard. Using bundled version for this
3. Added @arcana/auth and removed bundled sdk
4. Updated some config

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,31 +8,24 @@
     "serve": "vite preview"
   },
   "dependencies": {
+    "@arcana/auth": "^0.0.2",
     "@sentry/tracing": "^6.12.0",
     "@sentry/vue": "^6.12.0",
     "autoprefixer": "^10.3.4",
     "axios": "^0.21.1",
     "buffer": "^6.0.3",
```
Comment (ajithranka) [inline package.json:16]: Do we need to keep `buffer`?

File: src/main.js [hunk 2]
```diff
@@ -8,25 +8,19 @@ import "vue3-circle-progress/dist/circle-progress.css";
...
-//     new Integrations.BrowserTracing({
-//       routingInstrumentation: Sentry.vueRouterInstrumentation(router),
-//       tracingOrigins: sentryConfig.tracingOrigins,
-//     }),
-//   ],
-//   tracesSampleRate: 1.0,
-// });
+Sentry.init({
+  app,
+  dsn: sentryConfig.dsn,
```
Comment (ajithranka) [inline src/main.js:15]: The dsn value should default to `null` during `development`. Can we add a condition in `./utils/constants.js`?

[See this](https://github.com/getsentry/sentry-java/issues/574).

**Review by ajithranka — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (shrinathprabhu) [reply package.json:16]: No, I'll remove it

File: src/main.js (hunk 2 above)
Comment (shrinathprabhu) [reply src/main.js]: Check the updated code and let me know if this is fine

**Review by ajithranka — APPROVED** [review]

**Pull request merged**